news/_posts/2019-15-01-resilient-php-applications.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "Resilient PHP applications"
+subtitle: "Introducing the PrestaShop Circuit Breaker library"
+date:   2019-01-10 10:00:00
+authors: [ mandrieu ]
+icon: icon-blaster
+tags:
+ - 1.7
+---
+
+Using an extensible e-commerce platform like PrestaShop is awesome for both developers and e-merchants. Relying on an open source project with the possibility to add themes and plugins available in the store allows every business to create a shop that fits their needs.
+
+But, no matter how well designed is your project and how many tests you have, at some point your website can become unavailable.
```

Comment:
this sentence must be changed a bit in order to be the first sentence, so it's displayed on the article intro in the blog feed.

Reply:
Agree, do you have a suggestion? I'm not really good to market my content ^^

Reply:
```suggestion
No matter how well designed is your project and how many tests you have, at some point your website can become unavailable.
```
let's kick the first sentence, this is not really useful here, for the concept and the library is not only about online stores and open source.